### PR TITLE
Krishna profile update

### PR DESCRIPTION
**Pull request by Krishnapro**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2438"></a>

### CODE REVIEW

**Comment by Krishnapro** [comment]
@eddiejaoude  Here I have followed all the steps see the screenshot.

image


image

File: data/Krishnapro.json [hunk 1]
```diff
@@ -1,8 +1,18 @@
 {
   "name": "Krishna Kumar",
+  "displayStatsPublic": true,
   "type": "personal",
-  "bio": "Open Source & Cybersecurity Enthusiasts",
+  "bio": "Software Developer | Open Sorce | DevOps Enthusiasts",
```
Comment (amandamartin-dev) [inline data/Krishnapro.json:5]: You have one minor spelling error here you may want to adjust.  Open Sorce --> Open Source

Otherwise LGTM 🥳

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]

File: data/Krishnapro.json (hunk 1 above)
Comment (Krishnapro) [reply data/Krishnapro.json]: @amandamartin-dev  hey I fixed it, please check

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Mahesh-Kasabe — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Great collaboration. Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**